README.md

```diff
@@ -1,70 +1,166 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+## Hotel Room
```

Comment:
- [ ] Your app design is very nice.  Please, show it off with a screenshot so your users can see your app.

That way, your readme will look more professional 💪

Reply:
Thank you so much for the feedback, Miguel. I will handle this issue.